pages/profile/[username].js

```diff
@@ -162,6 +162,7 @@ function Profile() {
                                 <div className="text-center">
                                     <Image
                                         src={url}
+                                        alt="profile image"
```

Comment:
'User's profile image' can be more apt with name if name which we can generate from data in component

Reply:
`userRequest.user.username` for username or `userRequest.user.name` for name